doc/cookbook/source/examples/classifier/linear_svm.rst

```diff
@@ -0,0 +1,60 @@
+=============================
+Linear Support Vector Machine
+=============================
+
+Linear Support Vector Machine is a binary classifier which finds a hyper-plane such that the margins between the two classes are maximized. These margins can be described by the following equations:
+
+.. math::
+    {\bf w}^\top{\bf x} + b = 1
+
+    {\bf w}^\top{\bf x} + b = -1
+
+where :math:`{\bf w}` is vector of weights, :math:`{\bf x}` is feature vector and :math:`b` is bias.
+
+The distance between these two margins is :math:`\frac{2}{\|\mathbf{w}\|}`, so we want to minimize :math:`\|\mathbf{w}\|`. By using Lagrange Multipliers, we can show the solution to be:
+
+.. math::
+
+    \mathbf{w} = \sum_i \alpha_i y_i \mathbf{x}_i
+
+here most :math:`\alpha_i` turn out to be zero, which means that the solution is a sparse linear combination of the training features. The training features for which :math:`\alpha_i` are not zero are called support vectors.
+
+See Chapter 6 in :cite:`cristianini2000introduction` for a detailed introduction.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CBinaryLabels` as
+
+.. sgexample:: linear_svm.sg:create_features
+
+In order to run :sgclass:`CLibLinear`, we need to initialize some parameters like regularization coefficient and epsilon.
```

Comment:
C is coming from the loss function, so you can refer to it in math mode. `:math:`C`
epsilon is the residual convergence parameter of the solver, state that.
Also state that there are other solver types, referring to the class docs of CLibLinear